recipe/bld.bat

```diff
@@ -1,93 +1,77 @@
-@echo off
+@echo ON
 
-mkdir build
-cd build
-
-set CMAKE_CONFIG="Release"
+curl -L -O "https://github.com/opencv/opencv_contrib/archive/%PKG_VERSION%.tar.gz"
+%PYTHON% -c "import tarfile, os; with tarfile.open(os.environ['PKG_VERSION'] + '.tar.gz', 'r:gz'): tar.extractall()"
```

Comment:
So the `with` syntax isn't working. My guess is it just doesn't work on one line. Though if it does, it is missing `as tar` before the `:`.